src/components/NavBar/index.js

```diff
@@ -47,15 +57,37 @@ function NavBar() {
       }, 1000);
     });
 
+  // const [show, setshow] = useState(true);
+
+  // const handleClick = () => {
+  //   setshow(false);
+  // };
```

Comment:
Is it there for future help?

Reply:
No @Abhishek-kumar09  it was just for disposing the ad banner component I'll remove this :)

Reply:
@Abhishek-kumar09  ping!!